docs/ReleaseNotes.md

```diff
@@ -0,0 +1,77 @@
+# Antiplagiarism Release Notes
+This document provides information about fixes, enhancements, and key features that are available in Antiplagiarism.
+
+## Antiplagiarism 1.3
+Antiplagiarism 1.3 includes new features, enhancements, and bug fixes described in this section, as well as the features that have been implemented in the previous version.
+
+### Code Comparison Algorithm Enhancement
+Antiplagiarism now uses advanced code analysis algorithm from Roslyn which performs mich more complex comparison of syntax trees.
+This greatly improved the quality of analysis and allowed to find duplication even for small methods (about 5-10 lines).
+
+### Code Fragments in Data Grid
+The code fragments for the selected row can be shown directly in the data grid if you open the context menu with right click and select **Show Code**. 
+You can hide the code by right clicking on the selected row again and selecting **Hide Code** or by changing the selected row.
+
+### New Settings
+Antiplagiarism now has a new **Settings** panel. It can be expanded/collapsed by clicking on the button with a gear icon.
+
+#### Columns Visibility
+The **Column Visibility** button in the settings panel allows you to specify the columns visible in the data grid. 
+
+#### Show only rows exceeding threshold
+The **Show only rows exceeding threshold** button allows you to turn on mode in which all rows with **Similarity** value not exceeding **Threshold** are hidden.
+This feature can be combined with the filtering calculated results by the **Threshold** parameter.
+
+#### New Source Origin Modes
+In previous version of Antiplagiarism, you could only use the whole current solution as a source code for the comparison. Now you can choose one of the three modes:
+
+1. **Current Solution** to use the whole current solution for the comparison.
+2. **Selected Project** to use selected project from the current solution. The **Select Project** combobox will appear in which you can choose the project for comparison.
+3. **Selected Folder** to use selected folder with source code for comparison. You can specify the folder in the appeared **Select Folder** panel.
+
+#### New Supported Work Modes
+In previous version of Antiplagiarism, you could only specify the solution file as a reference solution for comparison. Now you can choose one of the three modes:
+
+1) **Self-Analysis** to compare the source code (selected with a consideration for specified **Source Origin**) with itself. This mode is useful for refactoring.
```

Comment:
I do not understand the text in parentheses

Reply:
We compare to sets of code. One is the source code we are working with. It is called "source code" or "source". The second is an example with which we compare which is called "reference" solution. 

The ""Work mode in general allows you to choose what to use as a reference - the current "source" itself for a self-analysis or some other solution for example, Acumatica sources.
The text in parenthesis specifies that the "source" will be used according to the  selected "Source Origin" mode.

The "Source Origin" mode allows you to choose what code you use as a "source". In the previous version you could use only the whole solution which was inconvenient in case of a big solution because you couldn't analyze only a part of it.
For example, if I wanted to analyze PX.Data I couldn't do it, I could only analyze NetTools or crreate another solution.